packages/design-system/src/components/__DEPRECATED__/text.tsx

```diff
@@ -1,6 +1,6 @@
 import { styled } from "../../stitches.config";
 
-export const __DEPRECATED__Text = styled("div", {
+export const DEPRECATEDText = styled("div", {
```

Comment:
hmm it would be nice to support `__DEPRECATED_Text__` btw I just noticed it was supposed to have the underscores like I this

Reply:
Otherwise if its impossible then I would just call it `DeprecatedText` camel case